CONTRIBUTING.md

```diff
@@ -0,0 +1,28 @@
+# Contributing
+
+👍🎉 First off, thanks for taking the time to contribute! 🎉👍
+
+Issues and pull requests are very welcome for bug reports and feature requests and/or proposals.
+
+Please provide as much information as possible, describing the bug or feature in detail. Screenshots and videos are great.
+
+All code needs to include programmatic tests, and to have been tested manually as well.
```

Comment:
Maybe mention that the tests in /test (where present in a repo) will be run on GitHub PR, and that the contributor may need to edit / add tests in that directory that adequately test new code.

Reply:
👍 agree we should mention CI, I've just included it. no need to mention tests location, these are standard in phoenix project and anybody contributing should be minimally versed on that